deepchecks/checks/integrity/mixed_data_types.py

```diff
@@ -126,22 +126,32 @@ def is_float(x) -> bool:
 
         return {'strings': strs_pct, 'numbers': nums_pct}
 
-    def add_condition_rare_type_ratio_not_less_than(self, max_rare_type_ratio: float = 0.01):
-        """Add condition - Whether the rarer data type (strings or numbers) have ratio higher than given ratio.
+    def add_condition_rare_type_ratio_not_in_range(self, ratio_range: Tuple[float, float] = (0.01, 0.1)):
+        """Add condition - Whether the ratio of rarer data type (strings or numbers) is not in the "danger zone".
+
+        The "danger zone" represents the following logic - if the rarer data type is, for example, 30% of the data,
+        than the column is presumable supposed to contain both numbers and numeric values. If the rarer data type is,
+        for example, less than 1% of the data, than it's presumably a contamination, but a negligible one. In the range
+        between, there is a real chance that the rarer data type may represent a problem to model training and
+        inference.
 
         Args:
-            max_rare_type_ratio (float): Minimal ratio allowed for the rarer type (numbers or strings)
+            ratio_range (Tuple[float, float]): The range between which the ratio of rarer data type in the column is
+                considered a problem.
```

Comment:
why not max and min thresholds? Just out of curiosity.

Reply:
mm think this is more compact